Source/WebGPU/WGSL/CompilationMessage.cpp

```diff
@@ -32,7 +32,7 @@ namespace WGSL {
 
 void CompilationMessage::dump(PrintStream& out) const
 {
-    out.print(m_span.m_line, ":", m_span.m_lineOffset, " length:", m_span.m_length, m_message);
+    out.print(m_span.m_line, ":", m_span.m_lineOffset, ":", m_message);
```

Comment:
Why?

Reply:
I came to this function to fix the mix colon, but then I didn't think the length was helpful, and it's a pretty standard error format to report only line and column. Happy to add it back though if you think it's helpful.